lib/vsc/zk/rsync/controller.py

```diff
@@ -66,17 +65,18 @@ def __init__(self, hosts, session=None, name=None, default_acl=None,
 
         super(RsyncController, self).__init__(**kwargs)
 
+        self.rsyncpath = rsyncpath.rstrip(os.path.sep)
+
         if not netcat:
-            rsyncpath = rsyncpath.rstrip(os.path.sep)
-            if not os.path.isdir(rsyncpath):
+            if verifypath and not self.basepath_ok():
                 self.log.raiseException('Path does not exists in filesystem: %s' % rsyncpath)
             if not os.path.isdir(self.RSDIR):
                 os.mkdir(self.RSDIR, 0700)
```

Comment:
make this `0o700` to make it clearer this is an octal number (required in python3)